waterlog/src/Components/AlertTable.js

```diff
@@ -26,31 +29,36 @@ class AlertTableComponent extends Component {
         <table>
           <tbody>
             {alerts.map((alert, index) => (
+              <Card className="card-alert">
+              <CardContent>
               <tr
                 key={index}
-                className="table-row table-row-unseen"
-                onClick={() =>
-                  (window.location.href = `alert/${alert.entityName}/${
+                className={alert.status === 1 ? "entity-disabled table-row" : "table-row table-row-unseen"}
+                onClick={alert.status === 2 ? () =>
+                (window.location.href = `alert/${alert.entityName}/${
                     alert.entityId
                   }`)
+                : () => {}
                 }
               >
                 <td className="leak-date">{formatDate(alert.date)}</td>
-                <td>{`${alert.entityName.toUpperCase()} ${
+                <td className="leak-name">{`${alert.entityName.toUpperCase()} ${
                   alert.entityId
                 } ${alert.entityType.toUpperCase()}`}</td>
-                <td>
-                  {alert.entityName === "Segment" ? `R${alert.cost}/hr` : ""}
+                <td className="leak-cost">
+                  {alert.entityName === "Segment" ? `R${(alert.cost).toFixed(2)}/hr` : ""}
```

Comment:
toFixed function will round the last number if the number after it is greater than 5.
Will your calculation still be accurate after a slight modification?